docs/dotnet/README.md

```diff
@@ -0,0 +1,22 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: .NET
+path_base_for_github_subdir:
+  from: content/en/docs/specs/semconv/dotnet/_index.md
```

Comment:
Can the `/en` language identifier be removed?

Reply:
@chalin do you happen to know if we need `/en` here or if we have some support for the user's content language on opentelemetry.io?

Reply:
The `from` value is a file system path, so you don't need to worry about the `en` there; and yes it is needed, because of the way the OTel website is setup.

While the website is ready for multilingual support, we only have `en` docs at the moment.